applications/ShapeOptimizationApplication/custom_utilities/mapping/mapper_vertex_morphing_matrix_free.h

```diff
@@ -401,6 +401,9 @@ class MapperVertexMorphingMatrixFree : public Mapper
         BuiltinTimer timer;
         KRATOS_INFO("ShapeOpt") << "Starting to update mapper..." << std::endl;
 
+        CreateListOfNodesInOriginModelPart();
```

Comment:
Putting those functions into the "update" is necessary because of the possible remeshing, am I seeing this right?

Reply:
yes. In remeshing, new nodes/elements are introduced.